Package.swift

```diff
@@ -40,6 +43,18 @@ let package = Package(
       name: "GoogleUtilities_Logger",
       dependencies: ["GoogleUtilities_Environment"],
       path: "GoogleUtilities/Logger",
-      publicHeadersPath: "Public")
+      publicHeadersPath: "Public"),
+    .target(
+      name: "FirebaseCore",
+      dependencies: ["GoogleUtilities_Environment", "GoogleUtilities_Logger"],
+      path: "Firebase/Core",
+      publicHeadersPath: "Public",
+      cSettings: [
+        .headerSearchPath("$(SRCROOT)/GoogleUtilities/Logger/Private"), // SPM doesn't support private headers
+        .define("FIRCore_VERSION", to: "0.0.1"),  // TODO Fix version
+        .define("Firebase_VERSION", to: "0.0.1"),  // TODO Fix version
+        .define("SWIFT_PACKAGE", to: "1"),  // SPM loses defaults if other cSettings
```

Comment:
This seems like a bug potentially, no?

Reply:
Yep. Almost every comment here is a bug or feature request.